Lib/uuid.py

```diff
@@ -700,24 +723,30 @@ def uuid3(namespace, name):
     """Generate a UUID from the MD5 hash of a namespace UUID and a name."""
     if isinstance(name, str):
         name = bytes(name, "utf-8")
-    from hashlib import md5
-    digest = md5(
-        namespace.bytes + name,
-        usedforsecurity=False
-    ).digest()
-    return UUID(bytes=digest[:16], version=3)
+    import _md5
```

Comment:
Could you go through `hashlib` here? `_md5` is a *fallback* implementation; the `usedforsecurity` argument is useless if you use `_md5` directly.

Reply:
After thinking a bit more, let's just revert to using `hashlib`. We don't lose too much and it would be more consistent with `uuid5`.